packages/salesforcedx-vscode-apex/package.json

```diff
@@ -74,6 +74,7 @@
     "lint:fix": "npm run lint -- --fix",
     "watch": "tsc -watch -p .",
     "clean": "shx rm -rf node_modules && cd out && node ../../../scripts/clean-all-but-jar.js && shx rm -rf coverage && shx rm -rf .nyc_output",
+    "copy:files": "shx cp -R ../../node_modules/@salesforce/apex-tmlanguage/grammars .",
```

Comment:
The copy:files task is already run post compile.

Reply:
note I checked the size of these files and they are very small so not a big concern for including in package. 712 bytes.